src/content/docs/infrastructure/amazon-integrations/get-started/introduction-aws-integrations.mdx

```diff
@@ -8,61 +8,126 @@ translate:
 * [AWS CloudFormation](https://github.com/newrelic/cloudformation-partner-integration)
 
 ## Integrations and AWS costs [#cloudwatch-billing]
 
-Keep in mind the following items:
-
-* AWS CloudWatch metric streams pricing is defined based on the number of metric updates. For up-to-date pricing information check [AWS CloudWatch Pricing](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html).
-* AWS Kinesis Data Firehose is used as the delivery method. For details, see the [AWS Firehose pricing page](https://aws.amazon.com/kinesis/data-firehose/pricing/).
-* AWS Config can be optionally enabled in your AWS account, and used to enrich CloudWatch metrics with resource metadata. With AWS Config, you are charged based on the number of configuration items recorded. See the [AWS Config pricing page](https://aws.amazon.com/config/pricing/) for details.
-* [AWS Resource Groups Tagging API](https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/overview.html) is used to enrich metrics with custom tags defined in the associated AWS resources (applicable to both polling and metric streams).
-* If polling integrations are enabled (instead of metric streams), New Relic uses the Amazon CloudWatch API to obtain metrics from the AWS services you monitor. The number of calls to the CloudWatch API increases as you enable more integrations. Add AWS resources to those integrations, or scale those integrations across more regions. This can cause requests to the CloudWatch API to exceed the <DNT>**1 million free**</DNT> limits granted by AWS and [increase your CloudWatch bill](/docs/infrastructure/amazon-integrations/troubleshooting/cloudwatch-billing-increase).
-* AWS offers enhanced monitoring for some of their services which allows for more metrics, more often. For example, see [RDS enhanced monitoring costs](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.overview.html#USER_Monitoring.OS.cost).
-
-## View your AWS data [#insights]
+This section directs you to relevant AWS docs so you can understand how our integrations affect your AWS billing. 
 
-Once you finish configuration, data from your AWS services reports directly to New Relic. AWS entities for many services are listed in our [entity explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer). Metrics and events appear in [metrics and events](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/). AWS data is also visible in the infrastructure UI.
+* The [AWS CloudWatch Pricing page](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html)defines AWS CloudWatch metric streams pricing based on the number of metric updates.
+* We sometimes use AWS Kinesis Data Firehose as the delivery method from AWS to New Relic, so we recommend reviewing the [AWS Firehose pricing page](https://aws.amazon.com/kinesis/data-firehose/pricing/).
```

Comment:
@akristen This is a little vague. As a reader, I think I'd like a little more context about when we (New Relic, I assume) use Firehose

Reply:
Hm, this might be on an integration-by-integration basis. Once I find a SME, I'll ask them to clarify this statement